socketcan_bridge/src/socketcan_bridge_nodelet.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Enway GmbH - All Rights reserved.
+ * Proprietary & confidential.
+ */
+#include "socketcan_bridge/socketcan_bridge_nodelet.h"
+
+#include <pluginlib/class_list_macros.h>
+
+PLUGINLIB_EXPORT_CLASS(socketcan_bridge::SocketcanBridgeNodelet, nodelet::Nodelet)
+
+socketcan_bridge::SocketcanBridgeNodelet::~SocketcanBridgeNodelet()
+{
+  driver_->shutdown();
+  driver_.reset();
+}
+
+void socketcan_bridge::SocketcanBridgeNodelet::onInit()
+{
+  ros::NodeHandle nh = getPrivateNodeHandle();
+  std::string can_device;
+  nh.param<std::string>("can_device", can_device, "can0");
+
+  driver_ = boost::make_shared<can::ThreadedSocketCANInterface> ();
```

Comment:
There shouldn't be a space before the parentheses

Reply:
fixed